salt/serializers/toml.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+"""
```

Comment:
Can you change all of your docstrings to single quotes? That's the style we use in salt.

Reply:
Done :-)
And if there is a place to find that error, could you please point it? (so I can deal with it later, to avoid any overhead on your side)